test.js

```diff
@@ -1,14 +1,24 @@
 import test from 'ava';
-import execa from 'execa';
+import {execa} from 'execa';
+
+const fixture = '\u001B[0m\u001B[4m\u001B[42m\u001B[31mfoo\u001B[39m\u001B[49m\u001B[24mfoo\u001B[0m';
 
 test('main', async t => {
-	const {stdout} = await execa('./cli.js', ['--version']);
-	t.true(stdout.length > 0);
+	const {stdout} = await execa('./cli.js', [fixture]);
+	t.is(stdout, 'foofoo');
 });
 
 test('stdin', async t => {
-	const {stdout} = await execa('./cli.js', {
-		input: '\u001B[0m\u001B[4m\u001B[42m\u001B[31mfoo\u001B[39m\u001B[49m\u001B[24mfoo\u001B[0m',
-	});
+	const {stdout} = await execa('./cli.js', {input: fixture});
 	t.is(stdout, 'foofoo');
 });
+
+test('no input', async t => {
+	/** @type {import('execa').ExecaError} */
+	const error = await t.throwsAsync(execa('./cli.js', {stdin: 'inherit'}));
+
+	t.like(error, {
+		stderr: 'Input required',
+		exitCode: 1,
+	});
```

Comment:
...yeah, I guess it makes sense that the action runner's shell isn't TTY lol. Does execa have a way to simulate it?

Reply:
No, but you can force Chalk to use colors even when not interactive using an environment variable.

Reply:
https://github.com/chalk/strip-ansi-cli/blob/5ed811e7dac35b2b1759eef570f73c1e63b82d64/cli.js#L24-L27

It’s for this check, I’m not sure if that would help.

Reply:
Ah, ok. You could use a PTY, but not worth the effort. Just skip that test.

Reply:
So this workaround worked: https://github.com/tommy-mitchell/strip-ansi-cli/commit/a8bb1594b9ed312af19f053d6e294819d88bd7d7

From https://github.com/actions/runner/issues/241#issuecomment-577360161